src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -144,6 +144,16 @@ struct SpatialRicci : db::SimpleTag {
   using type = tnsr::ii<DataType, Dim, Frame>;
 };
 
+/*!
+ * \brief Computes the scalar of the spatial Ricci tensor using the inverse
```

Comment:
This is a simple tag, it doesn't compute anything. `\brief Simple tag for the spatial Ricci scalar`